### PR TITLE
1.15.6 Release notes

### PR DESCRIPTION
**Pull request by dhawton**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.15.x/announcing-1.15.6/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Announcing Istio 1.15.6
+linktitle: 1.15.6
+subtitle: Patch Release
+description: Istio 1.15.6 patch release.
+publishdate: 2023-02-21
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.6/index.md:6]: Note adding time made in the 1.16 PR.

Comment (dhawton) [reply content/en/news/releases/1.15.x/announcing-1.15.6/index.md]: Done.

File: content/en/news/releases/1.15.x/announcing-1.15.6/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+linktitle: 1.15.6
+subtitle: Patch Release
+description: Istio 1.15.6 patch release.
+publishdate: 2023-02-21T07:00:00-06:00
+release: 1.15.6
+aliases:
+    - /news/announcing-1.15.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.15.5 and Istio 1.15.6.
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.6/index.md:12]: Nit: Should we remove the first sentence?

**Review by ericvn — APPROVED** [review]

File: content/en/news/releases/1.15.x/announcing-1.15.6/index.md (hunk 2 above)
Comment (dhawton) [reply content/en/news/releases/1.15.x/announcing-1.15.6/index.md:12]: I wasn't sure how to word the release note when there were no changes other than Go.

**Pull request merged**